pages/a/1k1-azure-post.html

```diff
@@ -0,0 +1,239 @@
+title: Crate guest post for Azure - Setting up Azure for Crate
+author: Chris Ward
+description: Setting up a 1001 node Crate cluster on Azure with Docker Swarm
+created: 2016-03-29
+status: publish
+post_type: post
+tags: azure, arm
+category: developernews
+
+# Crate guest post for Azure - Setting up Azure for Crate
+
+[Crate.io](https://crate.io/) is striving to create the most scalable database in our industry;
```

Comment:
"crate ... to create" always sounds funny. "is striving to be" would sound better to me.